pretix_servicefees/models.py

```diff
@@ -8,6 +8,6 @@ class ItemServicefeesSettings(models.Model):
     )
     exclude = models.BooleanField(
         verbose_name=_(
-            "Exclude this product from the calculation of per-ticket service fees"
+            "Exclude this product from the calculation of per-ticket and percentual service fees"
```

Comment:
it feels a bit weird to exclude from per-ticket *and* percentual fees here but only from per-ticket fees in the general Service Fee settings:
> Do not charge per-ticket service fee on non-admission products
> Do not charge per-ticket service fee on add-on products
> Do not charge per-ticket service fee on free products

For free products it doesn't matter obviously, though for add-on products it makes sense to only exclude them for per-ticket fee but not from the total...
So we can't change it there, so we probably just have to accept the weirdness...?

Reply:
Yup, I think we have to accept that :(